src/large_pages/node_large_page.cc

```diff
@@ -145,10 +145,10 @@ inline uintptr_t hugepage_align_down(uintptr_t addr) {
 #endif  // defined(__FreeBSD__)
 
 struct dl_iterate_params {
-  uintptr_t start;
-  uintptr_t end;
-  uintptr_t reference_sym;
-  std::string exename;
+  uintptr_t start = 0;
+  uintptr_t end = 0;
+  uintptr_t reference_sym = reinterpret_cast<uintptr_t>(&__node_text_start);
+  std::string exename = "";
```

Comment:
```suggestion
  std::string exename;
```
(it's superfluous for `std::string`.)